qemu/tests/cfg/nvdimm.cfg

```diff
@@ -0,0 +1,27 @@
+- nvdimm:
+    type = nvdimm
+    slots_mem = 4
+    maxmem_mem = 32G
+    kill_vm_on_error = yes
+    login_timeout = 240
+    only RHEL.7.3
```

Comment:
if only RHEL.7.3 here, how about 7.4? change the cfg again?
Or this case only support for 7.3? I do not think it's a good idea.